details.xml

```diff
@@ -0,0 +1,2 @@
+<dc:title>{title}</dc:title>
```

Comment:
We also won't need these then.

Reply:
Let me know if I have to remove this after checking my previous comment.